libs/packages/sam-formly/src/lib/formly-filters/advanced-filters/advanced-filters.component.ts

```diff
@@ -99,6 +108,10 @@ export class AdvancedFiltersComponent {
     }
   }
 
+  onShowInactiveChange(value){
+    this.showInactiveFiltersChange.emit(value);
+  }
+
   openDialog(): void {
```

Comment:
This openDialog method has gotten bit lengthy now with the new changes - minor suggestion here - maybe we can refactor the filedGroup field config and the inactiveGroup field config into their own small private methods - `getFiledGroup()` and `getInactiveGroup()`.

Reply:
Added 2 new getters and updated openDialog to make use of them